src/views/v2/emails/template-parts/body/tickets.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Event Tickets Emails: Main template > Body > Tickets.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/emails/template-parts/body/tickets.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link https://evnt.is/tickets-emails-tpl Help article for Tickets Emails template files.
+ *
+ * @version TBD
+ *
+ * @since TBD
+ *
+ * @var Tribe_Template  $this  Current template object.
+ */
+
+if ( empty( $tickets ) ) {
+	return;
+}
+$i = 0;
+
+$this->template( 'template-parts/body/tickets-total' );
+
+?>
+<tr>
+	<td style="padding:0;">
+		<table class="tec-tickets__email-table-content-tickets" role="presentation">
+		<?php foreach ( $tickets as $ticket ) : ?>
+			<?php $i++; ?>
```

Comment:
Should this not be at the end of the loop so we're not skipping the first ticket?

Reply:
The incremental var is just for display purposes, to use it in the "Ticket X of Y". Given that I defined it as `0`, I should do the `++` here to have the first element with `1`. It could be that or setting the var to `1` and moving the incremental to the end of the loop. It wouldn't change much though as we don't use it for anything else.

<img width="1442" alt="Screenshot 2023-02-09 at 10 05 31" src="https://user-images.githubusercontent.com/252415/217766494-84ef70b0-91f6-4cc0-a3ea-a405cdb7a6cd.png">

Reply:
Ah. My fault. I didn't look to see where it was being used. Thanks!